LEAF_Request_Portal/js/formSearchMultisite.js

```diff
@@ -0,0 +1,1111 @@
+        ) {
+          renderWidget(
+            i,
+            (function (widgetID, indicatorID, operator, match, gate) {
+              return function () {
+                $("#" + prefixID + "widgetIndicator_" + widgetID).val(
+                  indicatorID
+                );
+                $("#" + prefixID + "widgetIndicator_" + widgetID).trigger(
+                  "chosen:updated"
+                );
+                $("#" + prefixID + "widgetCod_" + widgetID).val(operator);
+                $("#" + prefixID + "widgetCod_" + widgetID).trigger("change");
+                $("#" + prefixID + "widgetCod_" + widgetID).trigger(
+                  "chosen:updated"
+                );
+                $("#" + prefixID + "widgetMat_" + widgetID).val(
+                  match.replace(/\*/g, "")
+                );
+                $("#" + prefixID + "widgetMat_" + widgetID).trigger(
+                  "chosen:updated"
+                );
+              };
+            })(
+              i,
+              advSearch[i].indicatorID,
+              advSearch[i].operator,
+              advSearch[i].match,
+              advSearch[i].gate
+            )
+          );
+        } else {
+          renderWidget(i);
+        }
+        $("#" + prefixID + "widgetCod_" + i).val(advSearch[i].operator);
+        if (typeof advSearch[i].match == "string") {
+          $("#" + prefixID + "widgetMat_" + i).val(
+            advSearch[i].match.replace(/\*/g, "")
+          );
+        }
+      }
+    }
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   * From: http://werxltd.com/wp/2010/05/13/javascript-implementation-of-javas-string-hashcode-method/
+   */
+  function getLocalStorageHash() {
+    var hash = 0,
+      i,
+      chr,
+      len;
+    if (document.URL.length == 0) return hash;
+    for (i = 0, len = document.URL.length; i < len; i++) {
+      chr = document.URL.charCodeAt(i);
+      hash = (hash << 5) - hash + chr;
+      hash |= 0; // Convert to 32bit integer
+    }
+    return hash;
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function setOrgchartPath(path) {
+    orgchartPath = path;
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function getLastSearch() {
+    return localStorage.getItem(localStorageNamespace + ".search");
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function setSearchFunc(func) {
+    searchFunc = func;
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function search(txt) {
+    if (txt != "*") {
+      localStorage.setItem(localStorageNamespace + ".search", txt);
+    }
+    return searchFunc(txt);
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function inputLoop() {
+    if ($("#" + prefixID + "searchtxt") == null) {
+      clearInterval(intervalID);
+      return false;
+    }
+    timer += timer > 5000 ? 0 : 200;
+    if (timer > 400) {
+      var txt = $("#" + prefixID + "searchtxt").val();
```

Comment:
There are a lot of vars in here and no lets.  Is this copied from other code?  We've been working on updating code to standards as we come across it.